src/layouts/Stats.js

```diff
@@ -1,7 +1,6 @@
-import React, { Component } from 'react'
-import { useTranslation, Trans } from 'react-i18next'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
 
-import AlignmentChart from '../components/AlignmentChart'
```

Comment:
Removed this import since this component is commented out from the code for now but Webpack was still including it.